src/PKSim.CLI.Core/Batch/Mapper/CompoundMapper.cs

```diff
@@ -181,7 +181,16 @@ private void setValue(ModelCompound compound, string alternativeName, string par
       {
          var alternative = compound.ParameterAlternativeGroup(alternativeName).DefaultAlternative;
          var parameter = alternative.Parameter(parameterName);
+         setValue(parameter, value);
+      }
+
+      private void setValue(IParameter parameter, double value)
```

Comment:
Defining this method in all mappers was key to get the expected snapshot after convesion. Most of this code will be removed once we have an agreement on all snapshot anyways